File: DAY1/rocketFuel.js
const fs = require('fs');

const calculateFuel = content => {
  const masses = content.split('\n');
  let totalFuel = 0;
  masses.forEach(mass => {
     totalFuel += Math.floor(mass / 3) - 2;
  });
  return totalFuel;
};

const main = () => {
  const content = fs.readFileSync('./DATA/mass.txt','utf8');
  console.log(calculateFuel(content));
};

main();

File: PY/DAY1/rocket_fuel.py
import math

def calc_fuel(content):
  mass = content.splitlines()
  total_fuel = 0
  for x in mass:
    total_fuel += math.floor(float(x) / 3) - 2
  return total_fuel  

f = open('mass.txt')
print(calc_fuel(f.read()))
f.close()